### PR TITLE
ADD deletePlot method and example

### PR DESCRIPTION
**Pull request by alexander-daniel**
### Will Resolve: https://github.com/plotly/plotly-nodejs/issues/18

@etpinard @bpostlethwaite 

What do you guys think? If y'all like it, I'll whip up some documentation for the new method!

### CODE REVIEW

**Comment by etpinard** [comment]
:dancer:

File: index.js [hunk 1]
```diff
@@ -236,6 +236,56 @@ Plotly.prototype.getImage = function (figure, opts, callback) {
...
+            'authorization': 'Basic ' + encodedAPIAuth
+        }
+    };
+
+    var req = https.request(options, function (res) {
+        parseRes(res, function (err, body) {
+
+            if (res.statusCode === 200) {
+
+                var msg = 'Successfully deleted plot: ' + self.username + ':' + fid;
```
Comment (bpostlethwaite) [inline index.js]: what is returned in the res on success?

**Comment by bpostlethwaite** [comment]
:dancer:

File: index.js (hunk 1 above)
Comment (alexander-daniel) [inline index.js]: the figure, I don't think we want to pass that through

Comment (bpostlethwaite) [inline index.js]: yeh lets give the user as much data as we can - they can ignore it

**Comment by alexander-daniel** [comment]
Yes, correct. I got the semantics mixed up, changing now!

**Comment by bpostlethwaite** [comment]
:dancer:

**Pull request merged**